docs/trace/extending-the-sdk/README.md

```diff
@@ -93,6 +93,12 @@ the library they instrument, and steps for enabling them.
   client](../../../src/OpenTelemetry.Instrumentation.StackExchangeRedis/README.md)
 * [SQL client](../../../src/OpenTelemetry.Instrumentation.SqlClient/README.md)
 
+More community contributed instrumentations are available in [OpenTelemetry .NET
```

Comment:
"More instrumentations are available in [OpenTelemetry .NET Contrib](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src)."